homeassistant/components/forked_daapd/const.py

```diff
@@ -0,0 +1,85 @@
+"""Const for forked-daapd."""
+from homeassistant.components.media_player.const import (
+    SUPPORT_CLEAR_PLAYLIST,
+    SUPPORT_NEXT_TRACK,
+    SUPPORT_PAUSE,
+    SUPPORT_PLAY,
+    SUPPORT_PLAY_MEDIA,
+    SUPPORT_PREVIOUS_TRACK,
+    SUPPORT_SEEK,
+    SUPPORT_SELECT_SOURCE,
+    SUPPORT_SHUFFLE_SET,
+    SUPPORT_STOP,
+    SUPPORT_TURN_OFF,
+    SUPPORT_TURN_ON,
+    SUPPORT_VOLUME_MUTE,
+    SUPPORT_VOLUME_SET,
+)
+
+CALLBACK_TIMEOUT = 8  # max time between command and callback from forked-daapd server
+CONF_LIBRESPOT_JAVA_PORT = "librespot_java_port"
+CONF_MAX_PLAYLISTS = "max_playlists"
+CONF_TTS_PAUSE_TIME = "tts_pause_time"
+CONF_TTS_VOLUME = "tts_volume"
+DEFAULT_PORT = 3689
+DEFAULT_SERVER_NAME = "My Server"
+DEFAULT_TTS_PAUSE_TIME = 1.2
+DEFAULT_TTS_VOLUME = 0.8
+DEFAULT_UNMUTE_VOLUME = 0.6
+DOMAIN = "forked_daapd"  # key for hass.data
+FD_NAME = "forked-daapd"
+HASS_DATA_REMOVE_LISTENERS_KEY = "REMOVE_LISTENERS"
+HASS_DATA_UPDATER_KEY = "UPDATER"
+KNOWN_PIPES = {"librespot-java"}
+PIPE_FUNCTION_MAP = {
+    "librespot-java": {
+        "async_media_play": "player_resume",
+        "async_media_pause": "player_pause",
+        "async_media_stop": "player_pause",
+        "async_media_previous_track": "player_prev",
+        "async_media_next_track": "player_next",
+    }
+}
+SIGNAL_ADD_ZONES = "forked-daapd_add_zones"
```

Comment:
Please add a string formatting argument specifier `{}` to the string so we can use `str.format` to add the entry id to the signal.